samples/word/01-basics/basic-api-call.yaml

```diff
@@ -1,7 +1,6 @@
-order: 1
```

Comment:
It would be great if we could get the TS basic api call at the top :) Also, I wonder if we should make the Basics folder smaller and move some of the snippets to other folders?